lib/matplotlib/tight_layout.py

```diff
@@ -123,6 +123,9 @@ def auto_adjust_subplotpars(fig, renderer,
     for subplots, ax_bbox, (num1, num2) in zip(subplot_list,
                                                ax_bbox_list,
                                                num1num2_list):
+        if all([not ax.get_visible() for ax in subplots]):
```

Comment:
Before making a PR, please check that your commit actually fixes the issue...ahem. :sweat_smile:

Skipping the laying calculations altogether (the approach here) meanwhile seems to work.